### ISSUE TITLE
Delete toast shows "undefined" when deleting a user in 2021-08-17

### ISSUE THREAD

**Issue by NoTuxNoBux** [keystonejs/keystone#6349]
# Bug report

## Describe the bug

The newly updated delete toast from 2021-08-17 shows `undefined` in its description when I delete a single user.

## To Reproduce

Steps to reproduce the behaviour. See below for a code snippet.

1. Go to Users.
2. Delete a user.
3. See the toast notification.

## Expected behaviour

According to the [release notes](https://github.com/keystonejs/keystone/releases/tag/2021-08-17), the toast should show some kin of display title for the deleted item, which is determined automatically.

## Screenshots

![afbeelding](https://user-images.githubusercontent.com/73220431/129696627-c7f864df-661c-4ca0-bb22-1cd575ebdbe2.png)

## System information

- OS: Linux
- Browser (if applies) Firefox 91

## Additional Context
My User configuration is mostly default, with a couple of added fields:

```ts
User: list({
    access: async (parameters) => {
        return checkPermissionsOmittedForBrevity(parameters);
    },
    ui: {
        listView: {
            initialColumns: ['name', 'email', 'group', 'anotherRelation', 'someRelation'],
        },
    },
    fields: {
        name: text({ isRequired: true }),
        email: text({ isRequired: true, isUnique: true }),
        password: password({ isRequired: true }),
        group: select({
            access: async (parameters) => {
                return checkFieldPermissionsOmittedForBrevity(parameters);
            },
            options: [
                { label: 'Group 1', value: 'group1' },
                { label: 'Group 2', value: 'group2' },
                { label: 'Group 3', value: 'group3' },
            ],
            ui: {
                displayMode: 'segmented-control',
            },
        }),
        anotherRelation: relationship({ ref: 'Bar.users', many: false }),
        someRelation: relationship({ ref: 'OtherEntity.user', many: false }),
    },
})
```

**Comment by raveling** [comment]
Thanks for posting this @NoTuxNoBux will discuss this tomorrow.

**Comment by raveling** [comment]
#6350 fixes this. Thanks @NoTuxNoBux for letting us know 🙌

**Comment by gwyneplaine** [comment]
@NoTuxNoBux fix for this is out now in [24.0.1](https://www.npmjs.com/package/@keystone-next/keystone), thanks again for creating the issue 🎉
